source/georef/street_network.h

```diff
@@ -130,6 +133,9 @@ struct PathFinder {
     /// Distance array for the Dijkstra
     std::vector<navitia::time_duration> distances;
 
+    /// Distance array for the Astar
+    std::vector<navitia::time_duration> costs;
```

Comment:
If benchmarking is good I would like to have a dedicated `PathFinder` as this is a big memory overhead

Reply:
Yep, and I won't have to override the init() method of PathFinder too.